langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/MistralAiModerationModel.java

```diff
@@ -0,0 +1,127 @@
+package dev.langchain4j.model.mistralai;
+
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.model.mistralai.internal.api.MistralAiModerationRequest;
+import dev.langchain4j.model.mistralai.internal.api.MistralAiModerationResponse;
+import dev.langchain4j.model.mistralai.internal.api.MistralModerationResult;
+import dev.langchain4j.model.mistralai.internal.client.MistralAiClient;
+import dev.langchain4j.model.moderation.Moderation;
+import dev.langchain4j.model.moderation.ModerationModel;
+import dev.langchain4j.model.output.Response;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static dev.langchain4j.internal.RetryUtils.withRetry;
+import static dev.langchain4j.internal.Utils.getOrDefault;
+import static java.util.Collections.singletonList;
+
+public class MistralAiModerationModel implements ModerationModel {
+
+    private final MistralAiClient client;
+    private final String modelName;
+    private final Integer maxRetries;
+
+    public MistralAiModerationModel(String baseUrl,
+                                    String apiKey,
+                                    Duration timeout,
+                                    Integer maxRetries,
+                                    String modelName,
+                                    Boolean logRequests,
+                                    Boolean logResponses) {
+
+        this.client = MistralAiClient.builder()
+            .baseUrl(getOrDefault(baseUrl, "https://api.mistral.ai/v1/moderations"))
```

Comment:
the default url seems incorrect

Reply:
It should probably be `https://api.mistral.ai/v1`, as in other models

Reply:
fixed